.travis.yml

```diff
@@ -12,8 +12,8 @@ env:
 addons:
   postgresql: '9.4'
 before_install:
-- rvm @global do gem uninstall bundler -a -x
-- rvm @global do gem install bundler -v 1.15.4
+  - gem install bundler -v 1.15.4
+  - alias bundle=bundle _1.15.4_
```

Comment:
@bdunne Does this need to be in quotes? 
`alias bundle="bundle _1.15.4_"`

Example: https://github.com/rspec/rspec-dev/blob/master/travis/.travis.yml#L12